source/exceptions.tex

```diff
@@ -114,10 +114,10 @@
   T1 t1;
   try {
     T2 t2;
-    if (@\textit{condition}@)
+    if (@\grammarterm{condition}@)
       goto lab;
-    } catch(...) { /* @\textit{handler 2}@ */ }
-  } catch(...) { /* @\textit{handler 1}@ */ }
+    } catch(...) { /* handler 2 */ }
```

Comment:
The following text refers to "handler 1" and "handler 2" in italics. If we make the text in the comments \tcode, should I adjust the following text to \tcode, too?